spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/pom.xml

```diff
@@ -17,6 +17,7 @@
 		<module>spring-cloud-gcp-integration-pubsub-sample-sender</module>
 		<module>spring-cloud-gcp-integration-pubsub-sample-receiver</module>
 		<module>spring-cloud-gcp-integration-pubsub-sample-test</module>
+    <module>spring-cloud-gcp-integration-pubsub-sample-json-payloads</module>
```

Comment:
Wrong indents: we use whitespaces only in the Docs. All other files must be with tabs.